spec/student_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rspec'
+require './student'
+
+describe Student do
+  before(:each) do
+    @student = Student.new(20, 'John', nil, 'Maths')
+  end
+  context 'when creating a new book' do
+    it 'should create a new student' do
+      student = Student.new(20, 'John', nil, 'Maths')
```

Comment:
- [ ] You're also repeating this piece of code, please use the variable you created in your `before` hook! 🤓